Last-Update: 2013-09-07
Forwarded: not-needed
Author: Dmitry Smirnov <onlyjob@member.fsf.org>
Bug-Debian: http://bugs.debian.org/722030
Bug-Abiword: http://bugzilla.abisource.com/13532
Description: fix FTBFS with evolution-data-server (>= 3.8)

--- a/src/text/ptbl/xp/pd_DocumentRDF.cpp
+++ b/src/text/ptbl/xp/pd_DocumentRDF.cpp
@@ -1883,9 +1883,9 @@
 
 
 #ifdef WITH_EVOLUTION_DATA_SERVER
 extern "C" {
-  #include <libebook/e-book.h>
+  #include <libebook/libebook.h>
 };
 
 static std::string get( EVCard* c, const char* v )
 {
