Author: Thaddaeus Tintenfisch <thad.fisch@gmail.com>
Description: Fix icon path in about dialog
Bug-Ubuntu: https://launchpad.net/bugs/1299582

--- a/src/af/xap/gtk/xap_UnixDlg_About.cpp
+++ b/src/af/xap/gtk/xap_UnixDlg_About.cpp
@@ -70,9 +70,7 @@ void XAP_UnixDialog_About::runModal(XAP_
 
 	// TODO Rob: use the more fancy "sidebar.png" logo, just like win32
 	if (!logo) {
-		std::string str (ICONDIR);
-		str += "/abiword.png";
-		logo = gdk_pixbuf_new_from_file (str.c_str(), NULL); // ignore errors
+		logo = gdk_pixbuf_new_from_file ("/usr/share/pixmaps/abiword.png", NULL); // ignore errors
 	}
 
 	dlg = gtk_about_dialog_new();
