From 9936beb5f7ddd2d68ddf5076af2ccba6949dedf3 Mon Sep 17 00:00:00 2001
From: Jari Aalto <jari.aalto@cante.net>
Date: Wed, 5 May 2010 01:28:35 +0300
Subject: [PATCH] abiword.keys: Add more MIME types
Organization: Private
Content-Type: text/plain; charset="utf-8"
Content-Transfer-Encoding: 8bit

Signed-off-by: Jari Aalto <jari.aalto@cante.net>
---
 abiword.keys |  104 ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 1 files changed, 104 insertions(+), 0 deletions(-)

diff --git a/abiword.keys b/abiword.keys
index 308384d..b098aed 100644
--- a/abiword.keys
+++ b/abiword.keys
@@ -9,3 +9,107 @@ application/msword:
 application/rtf:
 	open=abiword %f
 	view=abiword %f
+
+application/x-abicollab:
+	open=abiword %f
+	view=abiword %f
+
+application/x-crossmark:
+	open=abiword %f
+	view=abiword %f
+
+application/docbook+xml:
+	open=abiword %f
+	open=abiword %f
+
+application/vnd.oasis.opendocument.text:
+	open=abiword %f
+	open=abiword %f
+
+application/vnd.palm:
+	open=abiword %f
+	open=abiword %f
+
+application/vnd.plain:
+	open=abiword %f
+	open=abiword %f
+
+application/vnd.stardivision.writer:
+	open=abiword %f
+	open=abiword %f
+
+application/vnd.sun.xml.writer:
+	open=abiword %f
+	open=abiword %f
+
+application/vnd.wordperfect:
+	open=abiword %f
+	open=abiword %f
+
+application/wordperfect5.1:
+	open=abiword %f
+	open=abiword %f
+
+application/wordperfect6:
+	open=abiword %f
+	open=abiword %f
+
+application/xhtml+xml:
+	open=abiword %f
+	open=abiword %f
+
+application/x-applix-word:
+	open=abiword %f
+	open=abiword %f
+
+application/x-kword:
+	open=abiword %f
+	open=abiword %f
+
+application/x-mif:
+	open=abiword %f
+	open=abiword %f
+
+application/x-mswrite:
+	open=abiword %f
+	open=abiword %f
+
+application/x-palm-database:
+	open=abiword %f
+	open=abiword %f
+
+application/x-t602:
+	open=abiword %f
+	open=abiword %f
+
+text/abiword:
+	open=abiword %f
+	open=abiword %f
+
+text/html:
+	open=abiword %f
+	open=abiword %f
+
+text/plain:
+	open=abiword %f
+	open=abiword %f
+
+text/richtext:
+	open=abiword %f
+	open=abiword %f
+
+text/rtf:
+	open=abiword %f
+	open=abiword %f
+
+text/vnd.wap.wml:
+	open=abiword %f
+	open=abiword %f
+
+text/x-abiword:
+	open=abiword %f
+	open=abiword %f
+
+text/x-xml-abiword:
+	open=abiword %f
+	open=abiword %f
-- 
1.7.0

